docs/layouts/shortcodes/endpoint-description.md

```diff
@@ -37,9 +37,9 @@ Persistent Redis requires a disk to store data.
 You can define `<SERVICE_NAME>` as you like, but it shouldn't include underscores (`_`).
 {{ end }}
 
-Note that if you later change the name, it's treated as an entirely new service.
-This removes all data from your service.
-Always backup your data before changing the service.
+Note that if you change the name of the service,
+it triggers the creation of a new service and all existing data is lost.
```

Comment:
:lady_beetle: I think most of the changes here make sense, but `it triggers the creation` doesn't feel strongly enough as a replacement of the existing service. It seems like it might just create something new. I think getting more of the idea of the original database being replaced by the new one would help.